BoxContentSDK/BoxContentSDK/QueueManagers/BOXParallelAPIQueueManager.m

```diff
@@ -51,6 +52,10 @@ - (id)initWithSession:(BOXAbstractSession *)session
         _uploadsQueue.name = @"BOXParallelAPIQueueManager upload queue";
         _uploadsQueue.maxConcurrentOperationCount = 2;
 
+        _thumbnailDownloadsQueue = [[NSOperationQueue alloc] init];
+        _thumbnailDownloadsQueue.name = @"BOXParallelAPIQueueManager thumbnails downloads queue";
+        _thumbnailDownloadsQueue.maxConcurrentOperationCount = 10;
```

Comment:
10 seems quite high to me. I suspect you would just get throttled at a deeper network layer actually. Did you choose this based on tests on an actual device? Did you try 3G grade connections?